website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: big_card_for_md
+title: Apache Myriad 0.2.0 (Incubating) Released
+author: <a href="https://github.com/darinj">Darin Johnson</a> (Apache Myriad committer, 0.2.0 Release Manager)
+date:   2016-06-29 00:00:00
+categories: blogs
+---
+
+##### {{page.title}} 
+
+On behalf of the entire Myriad Team, I’m happy to announce the release of Apache Myriad 0.2.0 (incubating).  This is the 
```

Comment:
Yeah wasn't pleased but didn't want to fight it.  This is what Santosh did as well.